packages/flame/src/Flag/utils/FlagFactory.tsx

```diff
@@ -8,6 +8,7 @@ export type FlagProps = {
   /** Width of Flag. Height set automatically.  */
   size?: string;
   theme?: string;
+  className?: string;
```

Comment:
To follow what is already available for Icon?

Reply:
it rest props on an SVG, so yes, it's already available, just we never caught it due to having tests on a JS file, not TSX